recipes/rbc/LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Can you fix the sdist upstream by adding the license file location in `MANIFEST.in` ?

Reply:
Done in upstream.

Reply:
Can you remove this file then?

Reply:
Sure. Removed.